site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3331,6 +3332,21 @@ public function getRegistrationSections() {
         return $this->course_db->rows();
     }
 
+    /**
+     * Gets the default registration section for a given course and term from the courses table
+     */
+    public function getDefaultRegistrationSection(string $term, string $course): string|null {
+        $this->submitty_db->query("SELECT default_section_id FROM courses where term=? and course=?", [$term, $course]);
+        return $this->submitty_db->row()['default_section_id'] ?? null;
+    }
+
+    /**
+     * Updates the default registration section for self register courses.
+     */
+    public function setDefaultRegistrationSection(string $term, string $course, string $section_id): void {
+        $this->submitty_db->query("UPDATE courses set default_section_id=$section_id where term=? and course=?", [$term, $course]);
```

Comment:
SQL Injection possible here. Remove section_id from string and replace with ?.

Reply:
Fixed. Thanks